webpack.config.js

```diff
@@ -43,8 +43,8 @@ const img_loader = {
     },
 }
 
-const text_loader = {
-    test: /\.(eot|svg|ttf|woff|woff2)$/,
+const font_loader = {
+    test: /\.(eot|ttf|woff|woff2)$/,
     loader: 'file-loader?name=public/fonts/[name].[ext]'
```

Comment:
Use `outputPath` to specify folder(it should be relatieve to our public directory) where to put fonts and leave name as `[name].[ext]`